FSDigitaltechnik.tex

```diff
@@ -176,24 +175,30 @@ \section{Boolsche Algebra}
 		            & Boolesche Algebra                                     & Mengenalgebra                                         \\
 		            & $({0,1};\cdot , +, \overline{x})$                     & $(P(G);\cap , \cup, \overline{A};G,\emptyset )$       \\ \hline
 		Kommutativ  & $x \cdot y = y \cdot x$                               & $A \cap B = B \cap A$                                 \\
-		            & $x + y = y + x$                                       & $A \cup B = B \cup A$                                 \\
+		            & $x + y = y + x$                                       & $A \cup B = B \cup A$                                 \\ \lighthline
```

Comment:
Let's remove the `\lighthline`. I don't think they play very nice with the overlined variables

Reply:
As you want. But in the pdf version you can still see the overlines pretty well imo because  the ```
\lighthline
``` is much thinner than the actual `\hline`.
![image](https://github.com/latex4ei/Digitaltechnik/assets/38152878/f0a35ca3-8dcb-4686-9f91-3854e440b4ce)
Let me know if I should still remove it though

Reply:
For me it looks more readable without, but it is you that have to work with it so I leave the decision to you

Reply:
Yeah, I think it'll be fine like this because we will mostly only need the left column. There are no conflicts with `\overline` there and the `\lighthline` makes it easier to distinguish between the different rules imo.